docs/reference/indices/alias-exists.asciidoc

```diff
@@ -36,8 +36,6 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=index]
 [[alias-exists-api-query-params]]
 ==== {api-query-parms-title}
 
-include::{docdir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
-
```

Comment:
👍

Reply:
I'm not 100% sure. 
The mechanism is different, but the outcome the same (status 200 or 404).
If `allow_no_indices=false` and no index is matched then we get an exception at the index name resolution phase which is translated to 404.
If `allow_no_indices=true` and no index is matched then no aliases are found and this is again translated to 404.

Reply:
I think we should probably remove documentation for this parameter. It's not useful to the end user and the outcome will be the same regardless. Thanks for catching this @matriv.